WordPress/WordPressUITests/Tests/LoginTests.swift

```diff
@@ -119,9 +118,9 @@ class LoginTests: XCTestCase {
         XCTAssert(WelcomeScreen().isLoaded())
     }
 
-    // Unified email login fail
+    // Unified WordPress.com email login fail due to incorrect password
     // Replaces testUnsuccessfulLogin
-    func testWordPressUnsuccessfulLogin() {
+    func testWPcomInvalidPassword() {
```

Comment:
If the suggestion in the previous test makes sense, then this will keep the two consistent.

```suggestion
    func testWPComInvalidPassword() {
```